python/tk_houdini/ui_generation.py

```diff
@@ -134,6 +134,98 @@ def _itemNode(self, parent, label, id):
         node.text = id
         return item
 
+class AppCommandsPanels(AppCommandsUI):
+
```

Comment:
missing doc string for the class.

Also the plural form on both nouns is confusing to me! Wondering if it would make sense to call it something like `PanelHandler` or something?